colorbleed/maya/__init__.py

```diff
@@ -119,6 +126,9 @@ def on_open(_):
     from avalon.vendor.Qt import QtWidgets
     from ..widgets import popup
 
+    # Ensure scene's FPS is set to project config
+    lib.validate_fps()
```

Comment:
If this is checked using `before_save_check` do you still want to do it `on_save` too? Seems redundant.

Reply:
So you save a scene which has an invalid FPS set (which should possible as discussed yesterday), wouldn't you want to be reminded that the scene has been stored with an invalid FPS when opening? I can assume people (animators) will just work with it. This can result in timings which are wrong even when you update the keys to match the new animation which means extra work.

It is only displayed when the file being opened has been stored with the WRONG FPS. Yes it will be in your face until its fixed. This will be something that can be remedied with an override per shot if need be.